### ISSUE TITLE
groups creations

### ISSUE THREAD

**Issue by carlock2022** [traccar/traccar-web#1032]
Greetings, thanks for the excellent work, would it be possible for groups to be created automatically when creating a client? Or that the clients also appear in the Device search engine? that would allow a faster search for a device (or a client) currently the group can be done by hand, but it is repetitive and necessary if you wants to be able to easily search for someone's devices, another option would be for the clients to come in the devices tab and its equipment as a tree view, thanks for your time

**Comment by tananaev** [comment]
There's no concept of a "client". Do you mean a user? Why can't you just log in a the client to see their devices?

**Comment by carlock2022** [comment]
yes, user, sorry, my mistake, most admins leave a copy of all the devices in their account, so if for example the user "jesus christ" tells me that he has a problem with his equipment I have to look for him, (normally I should have created the group with that name to be able to see it) 

if I have to review, for example, 20 or 30 devices I have to be able to go from one to another easily (I repeat, if I create the groups I can search for them in the search engine) but in the current way If I don't create them, I have to enter the client's account, exit, go back to mine, go back to another client, (and repeat and repeat) 

it takes much more time than being able to look for it directly in my account and be able to go from one to another, ( That is why if the groups were created automatically or being able to search by client, it would greatly speed up the work of the admin) 

In this way, with a quick review, one can see the status of the clients and devices in general organized, in the current way, to see it easily, one has to remember the ID (normally the license plate) that the vehicles of each client have.

**Comment by tananaev** [comment]
Maybe you can provide more details on the use case. Why do you need to review multiple clients one by one?

**Comment by carlock2022** [comment]
Of course, let me tell you a little about my story so that you understand that I have come across these cases many times, I am not a programmer at all, but for 20 years I have been from installer to administrator of more than 7 companies, I have seen many platforms of tracing. I am trying to start my own life in this medium.

As a manager or administrator one has to ensure the proper functioning of the devices, the first thing one sees is the general list to know which ones are reporting or not, but today visually you do not know who each device belongs to,

The first line to know this is the users themselves, who communicate to say that they have not received an sms or email, so you have to check the web first, and send sms or status commands to see if it is a bug. gsm, or satellite, or devices, and of course wait for the response and return several times to the device

this is the common start of any day for an admin, depending on that he would coordinate the visits to the equipment, and even when he does, he will have to see the device several times to confirm that it went well,

When installing, the same thing happens, checking several times that it reports ok, that it emits alerts, that it has a good signal

Assuming you have a system of 1,000 vehicles and 10% of them require (albeit minimal) service, we would be servicing 100 vehicles per day,

not to mention that the equipment that you checked yesterday and the day before yesterday you check again today to confirm that they are fine and keep the user calm

It is more than common that when a user communicates with the administrator for 1 device, the two start to review all of them (and my wife's is good? and my mom's? and my cat's?)

All this means that not only several have to be reviewed, but many more since it is all about the user being functional, also in billing cases when charging one, all the others are reviewed.

with the current system, you either create the groups to be able to see all the users or you have to log in and log out (when you log out, you don't go back to the admin account, but you have to log out and back into the system of course) when review it's a matter of 2 seconds it gets long in the process of reloading and reloading the page,

that would be the most basic case, but there are many variants where a tree view or being able to search for them by users in the main tab helps a lot with system administration

**Comment by tananaev** [comment]
Based on the explanation it still sounds like you're working on a single client at a time, even if you need to check multiple devices they all belong to the same client. Maybe I misunderstood something.

**Comment by carlock2022** [comment]
I was putting 1 case, but (and it is daily) imagine that scenario with at least 10 users and 20 vehicles every morning. enter, exit, verify, device, exit, re-enter search for another user, verify device, return to the previous one and see if it worked, exit again, etc, (this added to the fact that the users tab is organized randomly) is solved a lot by create a group with each user and be able to search for it, but creating them is quite repetitive if you have 200 or 300 users, if the groups were created at the same time or you searched for the clients in the initial tab (where they are already device name , last connection and groups) everything would be much more fluid

**Comment by tananaev** [comment]
I guess it makes sense. Let's see what others think about it.

**Comment by carlock2022** [comment]
thank you very much for taking it into consideration

**Comment by mirindao** [comment]
users lack a search magnifying glass. groups missing search magnifying glass description and amount of device that the group has
